src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -502,6 +522,20 @@ public boolean onRunningTick(ItemStack aStack) {
         return true;
     }
 
+    private ItemStack getInputBusContent(int index) {
+        if (mItemCache[index] == NULL) {
+            ItemStack stuff = null;
+            if (index < mInputBusses.size()) {
+                GT_MetaTileEntity_Hatch_InputBus bus = mInputBusses.get(index);
+                if (isValidMetaTileEntity(bus)) {
+                    stuff = bus.getStackInSlot(0);
```

Comment:
What if the one which placed a bus doesn't have sorting enabled? This would practically get the AAL stuck. Wouldn't it be better to go with a while loop and find the first item one can?

Reply:
No. Go with first item only is the standard for assline like behavior. If this got changed, we must change the basic AL as well.